cos/static/game_page_api_calls.js

```diff
@@ -0,0 +1,13 @@
+//Gets information about players from the backend
+function get_player_info (cb_func,stage,layer) {
+    $.ajax({
+        url: '/api/game/getPlayersInGame',
+        type: 'POST',
+        dataType: 'json',
+        data: JSON.stringify({"game_id": "GAME1"}),
```

Comment:
Is GAME1 an intentional hardcode? After create and join game, I am storing the game ID in the following div:
<div id="game_id" hidden>{{project}}</div>

Maybe you can reference this?

Reply:
Yes, this was a temporary hardcode and it is fixed now.